### PR TITLE
feature(k8s): report operator versions to argus

### PR DESCRIPTION
**Pull request by fruch**
since we are going to be using operator much more
we should be start sending the versions to argus
so it would be visable for us investigating and for
the email template

Testing:
- [x] https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity-scylla-operator-3h-gke/2/
- [x] https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity-scylla-operator-3h-gke/5/

## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/tester.py [hunk 1]
```diff
@@ -420,6 +420,19 @@ def argus_collect_packages(self):
             self.log.info("Saving collected packages...")
             if len(versions) == 0:
                 self.argus_test_run.run_info.details.packages.append(self.generate_scylla_server_package())
+            if self.k8s_cluster:
```
Comment (vponomaryov) [inline sdcm/tester.py:423]: You forgot one more required field - the repo of the helm chart.

Comment (fruch) [reply sdcm/tester.py:423]: What do you mean ? do we have cases that we created a k8s cluster, but there's no helm ?

Or you mean we need that report as a package ?

Do we have more then one helm repo ?

Comment (vponomaryov) [reply sdcm/tester.py:423]: We have `3` `helm repos` where we get `scylla operator`, `scylla manager` and `scylla` `helm charts`:
- official stable/releases -> `https://storage.googleapis.com/scylla-operator-charts/stable`
- official latest non-releases -> `https://storage.googleapis.com/scylla-operator-charts/latest`
- dev -> `https://scylla-operator-charts-dev.storage.googleapis.com/latest`
And it is possible to use any other custom one, SCT won't block it.

That's why we have dedicated SCT config option for it: `SCT_K8S_SCYLLA_OPERATOR_HELM_REPO` / `k8s_scylla_operator_helm_repo`.

For debugging needs we must have it. It directly related to the understanding of the stuff that was used in a K8S test run.

Comment (fruch) [reply sdcm/tester.py:423]: so question is do have: 
* operator-image - docker image (scylladb/scylla-operator:tag)
* operator-helm-chart - k8s_scylla_operator_helm_repo + chart version

or:

* operator-image - docker image (scylladb/scylla-operator:tag)
* operator-helm-chart - chart version
* operator-helm-repo - k8s_scylla_operator_helm_repo

Comment (vponomaryov) [reply sdcm/tester.py:423]: Last one sounds to me much better.

Comment (fruch) [reply sdcm/tester.py]: so we'll settle with those

**Review by vponomaryov — APPROVED** [review]

**Pull request merged**